components/shared/IdentityPopover.vue

```diff
@@ -190,18 +205,11 @@ export default class IdentityPopover extends mixins(
       } else if (data?.firstMint?.nodes?.length > 0) {
         this.firstMintDate = data.firstMint.nodes[0].createdAt
       }
-
-      if (data?.collected?.nodes?.length > 0) {
-        this.lastBoughtDate = data.collected.nodes[0].collection.createdAt
-      } else if (data?.collected?.edges?.length > 0) {
-        this.lastBoughtDate = data.collected.edges[0].node.events[0].timestamp
-      }
```

Comment:
It would be better if we fetch the data from `userStatsByAccount`, so that we do not need to request twice for every identity. What's more, it could be cached.

If not, let's delete `lastBoughtDate` stuff in `userStatsByAccount.graphql`

Reply:
I tried using `userStatsByAccount` but it just wont give the correct format of data that is required for this

I have deleted the stuff from` userStatsByAccount.graphql `

Reply:
Ok. There is still something unused query in `@/queries/userStatsByAccount.graphql`. Let's delete it and it's good to go.

Reply:
Done!